arangod/Graph/EdgeDocumentToken.h

```diff
@@ -171,6 +172,23 @@ struct EdgeDocumentToken {
 #endif
 };
 }  // namespace graph
-
 }  // namespace arangodb
+
+namespace std {
+template<>
+  struct hash<arangodb::graph::EdgeDocumentToken> {
+  size_t operator()(arangodb::graph::EdgeDocumentToken const& value) const noexcept {
+    auto vslice = arangodb::velocypack::Slice(value.vpack());
```

Comment:
If you are on a coordinator, the use of `vpack()` is correct. If you are not a on a coordinator (e.g. DBServer), then you need to create a hash value out of a combination of `cid()` and  `localDocumentId()`.